confidence_cc/background_1_1_0_1.test

```diff
@@ -0,0 +1,28 @@
+1
+10 10 1 1
+1
+100
+
```

Comment:
shouldn't there be a coordinate associated with this image. like the starting location.

Reply:
Yes, it is encoded in the name of the test file. Existing test file format can handle arbitrary data.

Reply:
Just add it as an input and use that in the test.